frontend/src/scenes/plugins/pluginsLogic.ts

```diff
@@ -140,17 +140,6 @@ export const pluginsLogic = kea<pluginsLogicType>([
                     const response = await api.create(`api/organizations/@current/plugins/${id}/upgrade`)
                     capturePluginEvent(`plugin updated`, response)
                     actions.pluginUpdated(id)
-                    // Check if we need to update the config (e.g. new required field) and if so, open the drawer.
```

Comment:
this is useless - there can be several plugins that need to be updated, so at best this would fix things for this team only. One shouldn't update apps this way ever.